src/mmw/js/src/modeling/controllers.js

```diff
@@ -43,40 +44,65 @@ var ModelingController = {
                     project.getResultsIfNeeded();
                 });
         } else {
-            project = new models.ProjectModel({
-                name: 'Untitled Project',
-                created_at: Date.now(),
-                area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection()
-            });
+            if (App.currProject && App.singleProjectMode) {
+                project = App.currProject;
+                project.set('user_id', App.user.get('id'));
+                project.set('area_of_interest', App.map.get('areaOfInterest'));
+
+                // Clear current scenarios and start over.
+                // Must convert to an array first to avoid conflicts with
+                // collection events that are disassociating the model during
+                // the loop.
+                var locks = [];
+                _.each(project.get('scenarios').toArray(), function(model) {
+                    var $lock = $.Deferred();
+                    locks.push($lock);
+                    model.destroy({
```

Comment:
Since having once gone back from the `/project` view to `/analyze`, even if I make no changes, all the Scenarios are killed, I'm afraid that people will easily make mistakes and end up deleting a lot of work. I think we should not delete scenarios unless the AoI has changed, so if one goes from `/project` to `/analyze` to `/project` without making any changes, their old work is preserved.

Also, when resetting the AoI, the user should be shown a Confirm Modal that informs them of scenario deletion, and gives them a chance to cancel that.